samples/virtio/entropy/boards/xenvm_gicv3.overlay

```diff
@@ -0,0 +1,22 @@
+/{
+	soc {
+		virtio_mmio@a003e00 {
+			dma-coherent;
+			interrupt-parent = <&gic>;
+			interrupts = <0x00 0x2f 0x01 0x00>;
+			reg = <0x00 0xa003e00 0x00 0x200>;
+			compatible = "virtio,mmio";
+			status = "okay";
```

Comment:
If status is not disabled, I think it's "okay" by default. So we can safely remove it here.